sixtyfps_runtime/corelib/model.rs

```diff
@@ -376,8 +382,12 @@ impl Model for usize {
         *self
     }
 
-    fn row_data(&self, row: usize) -> Self::Data {
-        row as i32
+    fn row_data(&self, row: usize) -> Option<Self::Data> {
+        if row >= self.row_count() {
+            None
+        } else {
+            Some(row as i32)
+        }
```

Comment:
Not sure if we really need to return None in that case.
But if yes, may I suggest this shorter form. (can also be used for the one on bool)

```suggestion
        (row >= self.row_count()).then(|| row)
```